tutorials/3d/portals/tutorial_simple.rst

```diff
@@ -0,0 +1,83 @@
+Simple Tutorial
+===============
+Download this tutorial project:
+:download:`portals_tutorial_simple.zip <tutorial_simple/files/portals_tutorial_simple.zip>`.
+
+Unzip this project and add it to the Project Manager using the "Import" button.
+
+Introduction
+~~~~~~~~~~~~
+This tutorial will introduce you to building a "Hello World" room system with two rooms, and a portal in between.
+
+Step 1
+~~~~~~
+.. image:: tutorial_simple/img/tutorial_simple1.png
+
+- Create a new project
+- Add a :ref:`Spatial<class_Spatial>` as the scene root (I have called it 'Root')
+- Next add a :ref:`RoomManager<class_RoomManager>` node. We will need this later to process the room system.
+- Next we need to start defining our rooms. We create all our rooms under another Spatial we have called 'RoomList'.
+- Add a new :ref:`Room<class_Room>` node as a child of the roomlist.
+- Give it a  name that starts with the prefix ``Room_``. We add our chosen name as a suffix, here we have used 'kitchen'.
+- We will now create the geometry of our room. The names you give to the geometry is up to you.
+- Create a :ref:`MeshInstance<class_MeshInstance>` for the floor. Create a box mesh, and scale and position it to form a floor.
+- Create MeshInstances for the walls. Again use box meshes, and scale and position them, but be sure to leave an opening on one side (you will need to create two wall segments to do this on that side).
+
+Step 2
+~~~~~~
+.. image:: tutorial_simple/img/tutorial_simple2.png
+
+- Now we need to create the other room.
+- You can do this simply by duplicating the first room (select the 'Room_kitchen' node, right click and choose ``duplicate``).
+- Rotate and position the second room so that the openings line up.
+- Rename the second room to 'Room_lounge'.
```

Comment:
```suggestion
- Now we need to create the other room.
- You can do this simply by duplicating the first room (select the ``Room_kitchen`` node, right click and choose **Duplicate**).
- Rotate and position the second room so that the openings line up.
- Rename the second room to ``Room_lounge``.
```